source/installation-guide/packages-list.rst

```diff
@@ -321,3 +322,67 @@ MSU
 +===================+==============+=======================================================================================================================================================================================================================================+
 |  4.0.0 or later   |   32/64bits  | `msu-updates.json.gz <https://feed.wazuh.com/vulnerability-detector/windows/msu-updates.json.gz>`_ (`sha256 <https://feed.wazuh.com/vulnerability-detector/windows/msu-updates.meta>`__)                                              |
 +-------------------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Wazuh debug symbols
+-------------------
+
+Linux debug symbols
+^^^^^^^^^^^^^^^^^^^
+
+
++---------------+------------------------------------+--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Package type  |                 OS                 | Architecture | Packages                                                                                                                                                                                                                                                           |
```

Comment:
Could OS be aligned to the center?

Reply:
Hi @jnasselle 

I'm afraid I have not been able to find this option.

Reply:
Np!